libdokan/fs.go

```diff
@@ -272,8 +272,8 @@ func newSyntheticOpenContext() *openContext {
 func (f *FS) CreateFile(ctx context.Context, fi *dokan.FileInfo, cd *dokan.CreateData) (dokan.File, bool, error) {
 	// Only allow the current user access
 	if !fi.IsRequestorUserSidEqualTo(currentUserSID) {
-		f.log.Errorf("Refusing access: SID match error")
-		return nil, false, dokan.ErrAccessDenied
+		f.log.Errorf("FS CreateFile - Refusing real access: SID match error")
+		return openFakeRoot(ctx, f, fi)
```

Comment:
We're always guaranteed on Windows that the root will get a `CreateFile` call first on an empty string, before file listing is allowed or any other specific files can be looked up?